bakerydemo/templates/includes/card/picture-card.html

```diff
@@ -1,11 +1,15 @@
 {% load wagtailimages_tags %}
 
 <div class="picture-card">
-    <a class="picture-card__link" href="{{ childpage.url }}">
+    <a class="picture-card__link" href="{{ page.url }}">
         <figure class="picture-card__image">
-            {% image childpage.image fill-433x487-c100 %}
+            {% if portrait %}
+                {% image page.image fill-433x487-c100 %}
+            {% else %}
+                {% image page.image fill-645x480-c75 %}
```

Comment:
So far the designs just use two types of picture card, but it might be useful to specify width and height via variables some how - I'm guessing `fill-{{ width }}x{{ height }}` would cause an error with the syntax though